PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -1033,20 +1057,33 @@ struct HfTrackIndexSkimCreator {
     ccdb->setCaching(true);
     ccdb->setLocalObjectValidityChecking();
     lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbPathLut));
-    // if (!o2::base::GeometryManager::isGeometryLoaded()) {
-    //   ccdb->get<TGeoManager>(ccdbPathGeo);
-    // }
     runNumber = 0;
   }
 
+  /// Method to compute px, py, and px from the track parameters
+  /// \param trackPars are the track parameters
+  /// \param hfTrack1 is the momentum array to be filled
+  template <typename T, typename U>
+  void getPxPyPz(T const& trackPars, U& pVec)
+  {
+    auto pt = 1.f / std::abs(trackPars.getQ2Pt());
+    float cs = cosf(trackPars.getAlpha()), sn = sinf(trackPars.getAlpha());
+    auto r = std::sqrt((1.f - trackPars.getSnp()) * (1.f + trackPars.getSnp()));
+    pVec[0] = pt * (r * cs - trackPars.getSnp() * sn);
+    pVec[1] = pt * (trackPars.getSnp() * cs + r * sn);
+    pVec[2] = pt * trackPars.getTgl();
+  }
```

Comment:
This is a nice candidate for addition in [trackUtilities.h](https://github.com/AliceO2Group/O2Physics/blob/master/Common/Core/trackUtilities.h).

Reply:
Done!